src/shared/firewall-util-nft.c

```diff
@@ -34,7 +34,26 @@
 
 #define UDP_DPORT_OFFSET 2
 
-static sd_netlink_message **netlink_message_unref_many(sd_netlink_message **m) {
+/* for testing */
+static const char* nft_table_name(void) {
+        static const char *cached;
+
+        if (!cached)
+                cached = getenv("TEST_FIREWALL_UTIL_NFT_TABLE_NAME") ?: NFT_SYSTEMD_TABLE_NAME;
```

Comment:
doesn't matter, but i'd always use secure_getenv() unless we know for sure regular getenv() is fine and desirable.

Reply:
this codepath will run even outside of tests, no? hence I think prefixing the env var with SYSTEMD_ would be a good idea.